File: lib/src/main/java/com/timper/lib/di/feature/LibViewModel.java
package com.timper.lib.di.feature;

import com.timper.lonelysword.ActivityScope;
import com.timper.lonelysword.base.AppViewModel;

import javax.inject.Inject;

/**
 * User: tangpeng.yang
 * Date: 2019/3/20
 * Description:
 * FIXME
 */
@ActivityScope
public class LibViewModel extends AppViewModel {
    @Inject
    public LibViewModel() {
    }
}

File: lib/src/main/java/com/timper/lib/di/feature/LibViewModel.kt
package com.timper.lib.di.feature

import com.timper.lib.di.GetUseCaseUseCase
import com.timper.lonelysword.ActivityScope
import com.timper.lonelysword.base.AppViewModel
import javax.inject.Inject

/**
 * User:
 * Date:
 * Description:
 * FIXME
 */
@ActivityScope
class LibViewModel @Inject constructor(val getUseCaseUseCase: GetUseCaseUseCase) : AppViewModel() {

  override fun afterViews() {
    super.afterViews()
  }
}

